Engine/src/Engine/ResourceManagement/ShaderGenerator.h

```diff
@@ -11,14 +18,28 @@ namespace Engine
 	{
 	public:
 		ShaderGenerator(const char* vertexSource, const char* fragmentSource);
-		~ShaderGenerator();
 		int getProgramId() const;
 
 	private:
 		int m_programId;
 		int genShader(const char* source, GLenum shaderType);
 		bool requestStatus(GLuint shaderObj, GLenum status);
 	};
+
+	class ShaderNorms {
```

Comment:
should we have this class in a separate file?

Reply:
Done